tinker/admin/program_search/program_search_controller.py

```diff
@@ -33,7 +33,8 @@ def get_programs_for_dropdown(self, return_dict_for_renaming=False):
         else:
             program_concentrations = []
 
-        response = requests.get(app.config['PROGRAMS_XML'])
+        response = self.tinker_requests(app.config['PROGRAMS_XML'])
+        # response = requests.get(app.config['PROGRAMS_XML'])
```

Comment:
can you remove all of the commented out code?

Reply:
Removed